app/views/users/show.html.erb

```diff
@@ -15,9 +15,15 @@
 
     <div class='button_group'>
       <%= link_to 'See all posts', user_posts_path(user_id: @user.id), class: 'btn' %>
+      <%= link_to 'Create post', new_user_post_path(user_id: @user.id), class: 'btn' %>
       <%= link_to 'Back to main', users_path, class: 'back_btn' %>
     </div>
   <% else %>
     <p>No posts found.</p>
+
+    <div class='button_group'>
+      <%= link_to 'Create post', new_user_post_path(user_id: @user.id), class: 'btn' %>
+      <%= link_to 'Back to main', users_path, class: 'back_btn' %>
+    </div>
```

Comment:
- [ ] [OPTIONAL] It will be nice if there is a log out button for a user to able to logout :+1:

Reply:
@VICTOR-CHUKWU, I appreciate your review and will gladly take your suggestion into consideration. Thank you for your valuable feedback.